acceleration_wg.tf

```diff
@@ -4,11 +4,12 @@ locals {
     "vmayoral",
   ]
   acceleration_wg_repositories = [
+    "adaptive_component-release", 
     "ament_acceleration-release",
     "ament_vitis-release",
     "ros2acceleration-release",
     "tracetools_acceleration-release",
-    "vitis_common-release",
+    "vitis_common-release"
```

Comment:
We preserve all trailing commas in this project so that diffs only show added/removed lines and doesn't show formatting changes caused by adding or dropping a comma. I've updated this PR with e210de90ba2c99fe850288173f91f70aa6970c80 to preserve this comma.

Reply:
Oh, got it, thanks! :pray: